From 24bb20df088cae8e3b8e29fa17a083c31d6ab6ac Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Mon, 9 Dec 2013 19:10:48 +0000
Subject: [PATCH 2/6] tests (MinGW-w64): Fix build

Generalize two checks for _MSC_VER by replacing them
with checks for _WIN32.
---
 tests/glsl_optimizer_tests.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/glsl_optimizer_tests.cpp b/tests/glsl_optimizer_tests.cpp
index 7dab3d1..da8991f 100644
--- a/tests/glsl_optimizer_tests.cpp
+++ b/tests/glsl_optimizer_tests.cpp
@@ -14,7 +14,7 @@
 #if GOT_GFX
 
 // ---- Windows GL bits
-#ifdef _MSC_VER
+#ifdef _WIN32
 #define GOT_MORE_THAN_GLSL_120 1
 #include <windows.h>
 #include <gl/GL.h>
@@ -370,7 +370,7 @@ static StringVector GetFiles (const std::string& folder, const std::string& ends
 {
 	StringVector res;
 
-	#ifdef _MSC_VER
+	#ifdef _WIN32
 	WIN32_FIND_DATAA FindFileData;
 	HANDLE hFind = FindFirstFileA ((folder+"/*"+endsWith).c_str(), &FindFileData);
 	if (hFind == INVALID_HANDLE_VALUE)
-- 
2.4.5

